packages/relay/src/lib/constants.ts

```diff
@@ -175,4 +175,10 @@ export default {
     NEW_HEADS: 'newHeads',
     NEW_PENDING_TRANSACTIONS: 'newPendingTransactions',
   },
+
+  // @source: Foundry related constants below can be found at https://github.com/Arachnid/deterministic-deployment-proxy?tab=readme-ov-file#latest-outputs
```

Comment:
I think we want to remove references to "Foundry" as this can be relevant to others.